transformer_engine/jax/transformer.py

```diff
@@ -612,56 +647,59 @@ class TransformerLayer(nn.Module):
     layernorm_type : {'layernorm', 'rmsnorm'}, default = 'layernorm'
-        indicate the type of layer normalization.
+        Indicate the type of layer normalization.
     layernorm_epsilon: float, default = 1e-6
-        a value added to the denominator of layer normalization for numerical stability.
+        A value added to the denominator of layer normalization for numerical stability.
     hidden_dropout: float, default = 0.1
-        dropout probability for the dropout op after FC2 layer.
+        Dropout probability for the dropout op after FC2 layer.
     hidden_dropout_dims: Sequence[int], default = ()
-        dimensions that will share the same dropout mask for hidden
+        Dimensions that will share the same dropout mask for hidden
     attention_dropout: float, default = 0.1
-        dropout probability for the dropout op during multi-head attention.
+        Dropout probability for the dropout op during multi-head attention.
     dropout_rng_name: str, default = 'dropout'
-        the key in given RNGs via flax.linen.Module.apply that for
-        generate Dropout masks in the Multi-Head Attention.
+        The key in given RNGs via flax.linen.Module.apply that for
+        generating Dropout masks in the Multi-Head Attention.
     mha_kernel_init: Initializer, default =
         flax.linen.initializers.variance_scaling(1.0, 'fan_in', 'normal')
-        used for initializing weights of QKV and Output projection weights.
+        Used for initializing weights of QKV and Output projection weights.
+        It should be a callable object with three arguments (jax.random.PRNGKey, shape, dtype).
     mlp_kernel_init: Initializer, default =
         flax.linen.initializers.variance_scaling(1.0, 'fan_in', 'truncated_normal')
-        used for initializing weights of FC1 and FC2 layers.
+        Used for initializing weights of FC1 and FC2 layers.
+        It should be a callable object with three arguments (jax.random.PRNGKey, shape, dtype).
     mlp_activations: Sequence[str], default = ('relu', )
-        the sequence of activation functions to apply after the first linear transformation.
+        The sequence of activation functions to apply after the first linear transformation.
         Each activation has its own transformation layer.
     use_bias: bool, default = False
-        indicate whether to enable bias shifting for QKVO projections, FC1 and FC2.
-        if set to False, the layer will not learn additive biases.
+        Indicate whether to enable bias shifting for QKVO projections, FC1 and FC2.
+        If set to False, the layer will not learn additive biases.
     bias_init: Initializer, default = flax.linen.initializers.zeros
-        used for initializing bias of QKVO projections,
-        FC1 and FC2, only works when :attr:`use_bias=True`.
+        Used for initializing bias of QKVO projections,
+        FC1 and FC2. It only works when :attr:`use_bias=True`.
```

Comment:
```suggestion
        FC1 and FC2. It only used when :attr:`use_bias=True`.
```

Reply:
@mingxu1067 This pattern actually exists throughout, could you please go through all the cases here change "only works" -> "only used" so that it gives the correct picture?